ESP32 doesn't compile in master

used toolchain version:
```
xtensa-esp32-elf-gcc (crosstool-NG crosstool-ng-1.22.0-80-g6c4433a) 5.2.0
```

compilation error in samples/hello_world:
```
...
[ 30%] Building C object zephyr/CMakeFiles/zephyr.dir/arch/xtensa/soc/esp32/esp32-mp.c.obj
[ 31%] Building C object zephyr/CMakeFiles/zephyr.dir/arch/xtensa/core/cpu_idle.c.obj
[ 33%] Building C object zephyr/CMakeFiles/zephyr.dir/arch/xtensa/core/fatal.c.obj
[ 34%] Building ASM object zephyr/CMakeFiles/zephyr.dir/arch/xtensa/core/xtensa-asm2-util.S.obj
/home/matthias/workspace/zephyr/arch/xtensa/core/xtensa-asm2-util.S: Assembler messages:
/home/matthias/workspace/zephyr/arch/xtensa/core/xtensa-asm2-util.S:161: Error: unknown opcode or format name 'wsr.lbeg'
/home/matthias/workspace/zephyr/arch/xtensa/core/xtensa-asm2-util.S:163: Error: unknown opcode or format name 'wsr.lend'
/home/matthias/workspace/zephyr/arch/xtensa/core/xtensa-asm2-util.S:165: Error: unknown opcode or format name 'wsr.lcount'
/home/matthias/workspace/zephyr/arch/xtensa/core/xtensa-asm2-util.S:198: Error: unknown opcode or format name 'rsr.lbeg'
/home/matthias/workspace/zephyr/arch/xtensa/core/xtensa-asm2-util.S:198: Error: unknown opcode or format name 'rsr.lend'
/home/matthias/workspace/zephyr/arch/xtensa/core/xtensa-asm2-util.S:198: Error: unknown opcode or format name 'rsr.lcount'
zephyr/CMakeFiles/zephyr.dir/build.make:198: recipe for target 'zephyr/CMakeFiles/zephyr.dir/arch/xtensa/core/xtensa-asm2-util.S.obj' failed
make[2]: *** [zephyr/CMakeFiles/zephyr.dir/arch/xtensa/core/xtensa-asm2-util.S.obj] Error 1
CMakeFiles/Makefile2:521: recipe for target 'zephyr/CMakeFiles/zephyr.dir/all' failed
make[1]: *** [zephyr/CMakeFiles/zephyr.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2

```

--- a/arch/xtensa/soc/esp32/sdkconfig.h
+++ b/arch/xtensa/soc/esp32/sdkconfig.h
@@ -1,0 +1,5 @@
+/*
+ * This file is intentionally empty.  It is required to build Zephyr with
+ * esp-idf newer than git revision cb31222e.
+ */
+

